api/v1/storagecluster_types.go

```diff
@@ -377,6 +377,31 @@ type MultiCloudGatewaySpec struct {
 	// +nullable
 	// +optional
 	DisableLoadBalancerService bool `json:"disableLoadBalancerService,omitempty"`
+
+	// Allows Noobaa to connect to an external Postgres server
+	// +optional
+	ExternalPgConfig *ExternalPGSpec `json:"externalPgConfig,omitempty"`
+}
+
+type ExternalPGSpec struct {
+	// Enables Noobaa to use external Postgres
+	// +optional
+	Enable bool `json:"enable,omitempty"`
```

Comment:
Can we drop this field and assume that we need external postgres when `PGSecretName` and `PGSecretNamespace` are not nil?
I don't see why I would set `enable:false` and provide connection info or vice-versa.

Reply:
updated. i added the enable to have easy to read checks in future.